csharp/SparkCLR.linux.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
I just tried tweeking the .nuspec to use forward slash like here, and nuget pack on Windows seems to work ok.

Should mean can use the below text into the old .nuspec file and will work on both WIn + Linux.